src/backend/wasm.rs

```diff
@@ -101,17 +103,28 @@ impl<'a> WasmDialog<'a> {
             }
         };
 
-        let button = {
+        let ok_button = {
             let btn_el = document.create_element("button").unwrap();
             let btn: HtmlButtonElement = wasm_bindgen::JsCast::dyn_into(btn_el).unwrap();
 
-            btn.set_id("rfd-button");
+            btn.set_class_name("rfd-buttons");
```

Comment:
```suggestion
            btn.set_class_name("rfd-button");
```
Let's drop the `s`, perhaps it's just me but `rfd-buttons` suggests a container that contains buttons rather than a button.

Reply:
It's me hahah, and I agree with it!